src/router/index.js

```diff
@@ -20,14 +25,27 @@ export const checkTermsAccepted = (to, from, next) => {
   }
 };
 
```

Comment:
Previously, if you re-entered the terms route, you would have to agree again. This guard function redirects to the search route if the user has already agreed.